lib/maxwell/adapter/adapter.ex

```diff
@@ -30,6 +30,7 @@ defmodule Maxwell.Adapter do
         case res do
           %Conn{} -> res
           {:error, _reason, _conn} -> res
+
```

Comment:
Maybe remove this empty line to avoid this file being listed in the changed files?